File: Grass.java
import javax.swing.*;
public class Grass extends Plant{
    Grass(World world1, int x, int y) {
        SetStrength(0);
        SetInitiative(0);
        SetCords(x, y);
        SetMove(true);
        world = world1;
    }
    @Override
    public String getName() {

        return "GRASS";
    }
    @Override
    public void Drawing() {
        ImageIcon icon = new ImageIcon("src/Icons/grass.png");
        world.GetButton(GetCords()).setIcon(icon);
    }
}

File: grass.py
from plant import Plant

class Grass(Plant):
    def __init__(self, world1, x, y):
        super().__init__()
        self.SetStrength(0)
        self.SetInitiative(0)
        self.SetCords(x, y)
        self.SetMove(True)
        self.world = world1

    def getName(self):
        return "GRASS"

    def Drawing(self):
        self.world.buttons[self.GetCords().y -1][self.GetCords().x -1].configure(bg='#800080')
    
    def New(self, text, reference, x, y):
        self.world.GetOrganisms().add(Grass(reference, x, y))    
